src/compressAlignment.hpp

```diff
@@ -77,26 +77,32 @@ that can be represented                    3       7      511   -2^19   -2^31
 
 // Standard library.
 #include "string.hpp"
-#include "span.hpp"
 
 
 namespace shasta {
     void compress(const Alignment&, string&);
-    void decompress(span<const char>, Alignment);
+    void decompress(const string&, Alignment&);
+
+    void testAlignmentCompression();
 
     namespace compressAlignment {
         class Format0;
         class Format1;
         class Format2;
         class Format3;
         class Format4;
+
+        uint8_t extractFormatIdentifier(const char);
     }
 }
 
 
 
 class shasta::compressAlignment::Format0 {
 public:
+    static const uint8_t ID = 0b0;
```

Comment:
Any special reason to use the unusual bit format for this constant? How about just `0`?

Reply:
It increases readability of what the expected format identifying bits are. Of course, that's a personal preference. I don't feel strongly about it.

Reply:
Either way is fine with me. Just curious.